### PR TITLE
Export2Pdf: optional grid layer

### PR DESCRIPTION
**Pull request by thiloSchlemmer**

### CODE REVIEW

File: test/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.test.js [hunk 1]
```diff
@@ -386,14 +416,36 @@ describe('ExportMfpToolContent', () => {
 
 			const toggleButton = element.shadowRoot.querySelector('#autorotation');
 
-			toggleButton.click();
+			toggleButton.dispatchEvent(getToggleEventFor(toggleButton));
```
Comment (taulinger) [inline test/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.test.js:419]: What's the cause of this workaround?

File: test/modules/olMap/formats/Mfp3Encoder.test.js [hunk 2]
```diff
@@ -2031,4 +2044,29 @@ describe('BvvMfp3Encoder', () => {
...
+
+
+			// act & assert
+			validScales.forEach((validScale, index) => {
+				const actualGridLayerSpec = classUnderTest._encodeGridLayer(validScale);
+				const expectedSpacing = expectedSpacings[index];
+				expect(actualGridLayerSpec.spacing).toEqual([expectedSpacing, expectedSpacing]);
+			});
+		});
+		it('uses the default spacing for a unknown scale', () => {
```
Comment (taulinger) [inline test/modules/olMap/formats/Mfp3Encoder.test.js:2062]: missing new line

**Review by taulinger — COMMENTED** [review]
Excellent, just some questions/suggestions...

Did you already discuss the color of the grid?

**Comment by thiloSchlemmer** [comment]
> Excellent, just some questions/suggestions...
> 
> Did you already discuss the color of the grid? grafik

No, but just leaved it by the default one and otherwise setting explicitly the label color to black ...
https://github.com/ldbv-by/bav4-nomigration/blob/7c88fa01751ad285c1a659524b1361e56c874069/src/modules/olMap/services/Mfp3Encoder.js#L787

I will investigate this unexpected color

File: test/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.test.js (hunk 1 above)
Comment (thiloSchlemmer) [reply test/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.test.js:419]: The 'toggleButton' was a Checkbox component before, which have a click-action.
Now the 'toggleButton' is a Toggle component, which does not register a EventListener for 'click'-events.

File: src/modules/toolbox/i18n/toolbox.provider.js [hunk 3]
```diff
@@ -204,9 +206,11 @@ export const provide = (lang) => {
 				toolbox_exportMfp_id_a4_portrait: 'A4 Hochformat',
 				toolbox_exportMfp_id_a3_landscape: 'A3 Querformat',
 				toolbox_exportMfp_id_a3_portrait: 'A3 Hochformat',
-				toolbox_exportMfp_options: 'Optionen',
+				toolbox_exportMfp_options: 'Exportoptionen',
```
Comment (taulinger) [inline src/modules/toolbox/i18n/toolbox.provider.js:209]: Export-Optionen (?)

File: src/modules/toolbox/i18n/toolbox.provider.js [hunk 4]
```diff
@@ -100,9 +100,11 @@ export const provide = (lang) => {
 				toolbox_exportMfp_id_a4_portrait: 'A4 portrait',
 				toolbox_exportMfp_id_a3_landscape: 'A3 landscape',
 				toolbox_exportMfp_id_a3_portrait: 'A3 portrait',
-				toolbox_exportMfp_options: 'Options',
+				toolbox_exportMfp_options: 'Exportoptions',
```
Comment (taulinger) [inline src/modules/toolbox/i18n/toolbox.provider.js:103]: Export options (?)

**Review by taulinger — APPROVED** [review]
Competently done!

**Review by taulinger — APPROVED** [review]
Thx!

**Pull request merged**